Stripe/STPCard.m

```diff
@@ -135,6 +148,35 @@ - (BOOL)isEqualToCard:(STPCard *)other {
     return [self.cardId isEqualToString:other.cardId];
 }
 
+- (NSString *)description {
```

Comment:
Should this be `debugDescription` instead? I think `description` is supposed to be potentially user-facing text theoretically.